NPE in FontConfiguration.getVersion

I am using AutoSize Columns and getting this in Native mode.

**XLSX:** 
```
2023-02-22 16:28:44,535 SEVERE [org.pri.app.exc.PrimeExceptionHandler] (executor-thread-5) java.lang.reflect.InvocationTargetException: java.lang.InternalError: java.lang.reflect.InvocationTargetException
        at java.desktop@17.0.6/sun.font.FontManagerFactory$1.run(FontManagerFactory.java:87)
        at java.base@17.0.6/java.security.AccessController.executePrivileged(AccessController.java:168)
        at java.base@17.0.6/java.security.AccessController.doPrivileged(AccessController.java:318)
        at java.desktop@17.0.6/sun.font.FontManagerFactory.getInstance(FontManagerFactory.java:75)
        at java.desktop@17.0.6/java.awt.Font.getFont2D(Font.java:526)
        at java.desktop@17.0.6/java.awt.Font.canDisplayUpTo(Font.java:2282)
        at java.desktop@17.0.6/java.awt.font.TextLayout.singleFont(TextLayout.java:469)
        at java.desktop@17.0.6/java.awt.font.TextLayout.<init>(TextLayout.java:530)
        at org.apache.poi.ss.util.SheetUtil.getCellWidth(SheetUtil.java:224)
        at org.apache.poi.ss.util.SheetUtil.getCellWidth(SheetUtil.java:185)
        at org.apache.poi.ss.util.SheetUtil.getColumnWidthForRow(SheetUtil.java:336)
        at org.apache.poi.ss.util.SheetUtil.getColumnWidth(SheetUtil.java:280)
        at org.apache.poi.ss.util.SheetUtil.getColumnWidth(SheetUtil.java:257)
        at org.apache.poi.xssf.streaming.SXSSFSheet.autoSizeColumn(SXSSFSheet.java:1621)
        at org.apache.poi.xssf.streaming.SXSSFSheet.autoSizeColumn(SXSSFSheet.java:1567)
 at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:775)
        at org.graalvm.nativeimage.builder/com.oracle.svm.core.posix.thread.PosixPlatformThreads.pthreadStartRoutine(PosixPlatformThreads.java:203)
Caused by: java.lang.reflect.InvocationTargetException
        at java.base@17.0.6/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
        at java.base@17.0.6/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
        at java.desktop@17.0.6/sun.font.FontManagerFactory$1.run(FontManagerFactory.java:85)
        ... 92 more
Caused by: java.lang.NullPointerException
        at java.desktop@17.0.6/sun.awt.FontConfiguration.getVersion(FontConfiguration.java:1264)
        at java.desktop@17.0.6/sun.awt.FontConfiguration.readFontConfigFile(FontConfiguration.java:224)
        at java.desktop@17.0.6/sun.awt.FontConfiguration.init(FontConfiguration.java:106)
        at java.desktop@17.0.6/sun.awt.X11FontManager.createFontConfiguration(X11FontManager.java:706)
        at java.desktop@17.0.6/sun.font.SunFontManager$2.run(SunFontManager.java:358)
        at java.desktop@17.0.6/sun.font.SunFontManager$2.run(SunFontManager.java:315)
        at java.base@17.0.6/java.security.AccessController.executePrivileged(AccessController.java:168)
        at java.base@17.0.6/java.security.AccessController.doPrivileged(AccessController.java:318)
        at java.desktop@17.0.6/sun.font.SunFontManager.<init>(SunFontManager.java:315)
        at java.desktop@17.0.6/sun.awt.FcFontManager.<init>(FcFontManager.java:35)
        at java.desktop@17.0.6/sun.awt.X11FontManager.<init>(X11FontManager.java:56)
```

**XLS:**
```
2023-02-22 16:31:07,717 SEVERE [org.pri.app.exc.PrimeExceptionHandler] (executor-thread-5) java.lang.reflect.InvocationTargetException: java.lang.InternalError: java.lang.reflect.InvocationTargetException
        at java.desktop@17.0.6/sun.font.FontManagerFactory$1.run(FontManagerFactory.java:87)
        at java.base@17.0.6/java.security.AccessController.executePrivileged(AccessController.java:168)
        at java.base@17.0.6/java.security.AccessController.doPrivileged(AccessController.java:318)
        at java.desktop@17.0.6/sun.font.FontManagerFactory.getInstance(FontManagerFactory.java:75)
        at java.desktop@17.0.6/java.awt.Font.getFont2D(Font.java:526)
        at java.desktop@17.0.6/java.awt.Font.canDisplayUpTo(Font.java:2282)
        at java.desktop@17.0.6/java.awt.font.TextLayout.singleFont(TextLayout.java:469)
        at java.desktop@17.0.6/java.awt.font.TextLayout.<init>(TextLayout.java:530)
        at org.apache.poi.ss.util.SheetUtil.getCellWidth(SheetUtil.java:224)
        at org.apache.poi.ss.util.SheetUtil.getCellWidth(SheetUtil.java:185)
        at org.apache.poi.ss.util.SheetUtil.getColumnWidthForRow(SheetUtil.java:336)
        at org.apache.poi.ss.util.SheetUtil.getColumnWidth(SheetUtil.java:280)
        at org.apache.poi.ss.util.SheetUtil.getColumnWidth(SheetUtil.java:257)
        at org.apache.poi.hssf.usermodel.HSSFSheet.autoSizeColumn(HSSFSheet.java:2298)
        at org.apache.poi.hssf.usermodel.HSSFSheet.autoSizeColumn(HSSFSheet.java:2280)
 at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:775)
        at org.graalvm.nativeimage.builder/com.oracle.svm.core.posix.thread.PosixPlatformThreads.pthreadStartRoutine(PosixPlatformThreads.java:203)
Caused by: java.lang.reflect.InvocationTargetException
        at java.base@17.0.6/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
        at java.base@17.0.6/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
        at java.desktop@17.0.6/sun.font.FontManagerFactory$1.run(FontManagerFactory.java:85)
        ... 92 more
Caused by: java.lang.NullPointerException
        at java.desktop@17.0.6/sun.awt.FontConfiguration.getVersion(FontConfiguration.java:1264)
        at java.desktop@17.0.6/sun.awt.FontConfiguration.readFontConfigFile(FontConfiguration.java:224)
        at java.desktop@17.0.6/sun.awt.FontConfiguration.init(FontConfiguration.java:106)
        at java.desktop@17.0.6/sun.awt.X11FontManager.createFontConfiguration(X11FontManager.java:706)
        at java.desktop@17.0.6/sun.font.SunFontManager$2.run(SunFontManager.java:358)
        at java.desktop@17.0.6/sun.font.SunFontManager$2.run(SunFontManager.java:315)
        at java.base@17.0.6/java.security.AccessController.executePrivileged(AccessController.java:168)
        at java.base@17.0.6/java.security.AccessController.doPrivileged(AccessController.java:318)
        at java.desktop@17.0.6/sun.font.SunFontManager.<init>(SunFontManager.java:315)
        at java.desktop@17.0.6/sun.awt.FcFontManager.<init>(FcFontManager.java:35)
        at java.desktop@17.0.6/sun.awt.X11FontManager.<init>(X11FontManager.java:56)
```

OK I reproduced it with a real docker container. Steps to reproduce:

1. Check out my branch.
2. Run `mvn -Pnative` to build the native image and install it in Docker
3. Once built run the Docker image `docker run -i --rm -p 8080:8080 quarkus/quarkus-poi:latest`
4. In your browser go to URL `http://localhost:8080/poi/xlxs`
5. See stacktrace...

```
__  ____  __  _____   ___  __ ____  ______
 --/ __ \/ / / / _ | / _ \/ //_/ / / / __/
 -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \
--\___\_\____/_/ |_/_/|_/_/|_|\____/___/
2023-02-22 19:38:00,560 INFO  [io.net.uti.int.PlatformDependent] (Thread-1) Your platform does not provide complete low-level API for accessing direct buffers reliably. Unless explicitly requested, heap buffer will always be preferred to avoid potential system instability.
2023-02-22 19:38:00,576 INFO  [io.quarkus] (main) quarkus-poi-integration-tests 999-SNAPSHOT native (powered by Quarkus 3.0.0.Alpha4) started in 0.027s. Listening on: http://0.0.0.0:8080
2023-02-22 19:38:00,576 INFO  [io.quarkus] (main) Profile prod activated.
2023-02-22 19:38:00,576 INFO  [io.quarkus] (main) Installed features: [awt, cdi, poi, resteasy, smallrye-context-propagation, vertx]
2023-02-22 19:38:08,369 ERROR [io.qua.ver.htt.run.QuarkusErrorHandler] (executor-thread-0) HTTP Request to /poi/xlxs failed, error id: 8acb0117-f378-4416-8c67-f3288abeebe0-1: org.jboss.resteasy.spi.UnhandledException: java.lang.InternalError: java.lang.reflect.InvocationTargetException
        at org.jboss.resteasy.core.ExceptionHandler.handleApplicationException(ExceptionHandler.java:110)
        at org.jboss.resteasy.core.ExceptionHandler.handleException(ExceptionHandler.java:370)
        at org.jboss.resteasy.core.SynchronousDispatcher.writeException(SynchronousDispatcher.java:218)
        at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:519)
        at org.jboss.resteasy.core.SynchronousDispatcher.lambda$invoke$4(SynchronousDispatcher.java:261)
        at org.jboss.resteasy.core.SynchronousDispatcher.lambda$preprocess$0(SynchronousDispatcher.java:161)
        at org.jboss.resteasy.core.interception.jaxrs.PreMatchContainerRequestContext.filter(PreMatchContainerRequestContext.java:364)
        at org.jboss.resteasy.core.SynchronousDispatcher.preprocess(SynchronousDispatcher.java:164)
        at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:247)
        at io.quarkus.resteasy.runtime.standalone.RequestDispatcher.service(RequestDispatcher.java:82)
        at io.quarkus.resteasy.runtime.standalone.VertxRequestHandler.dispatch(VertxRequestHandler.java:147)
        at io.quarkus.resteasy.runtime.standalone.VertxRequestHandler$1.run(VertxRequestHandler.java:93)
        at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:576)
        at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
        at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
        at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
        at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.base@17.0.6/java.lang.Thread.run(Thread.java:833)
        at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:775)
        at org.graalvm.nativeimage.builder/com.oracle.svm.core.posix.thread.PosixPlatformThreads.pthreadStartRoutine(PosixPlatformThreads.java:203)
Caused by: java.lang.InternalError: java.lang.reflect.InvocationTargetException
        at java.desktop@17.0.6/sun.font.FontManagerFactory$1.run(FontManagerFactory.java:87)
        at java.base@17.0.6/java.security.AccessController.executePrivileged(AccessController.java:168)
        at java.base@17.0.6/java.security.AccessController.doPrivileged(AccessController.java:318)
        at java.desktop@17.0.6/sun.font.FontManagerFactory.getInstance(FontManagerFactory.java:75)
        at java.desktop@17.0.6/java.awt.Font.getFont2D(Font.java:526)
        at java.desktop@17.0.6/java.awt.Font.canDisplayUpTo(Font.java:2282)
        at java.desktop@17.0.6/java.awt.font.TextLayout.singleFont(TextLayout.java:469)
        at java.desktop@17.0.6/java.awt.font.TextLayout.<init>(TextLayout.java:530)
        at org.apache.poi.ss.util.SheetUtil.getCellWidth(SheetUtil.java:224)
        at org.apache.poi.ss.util.SheetUtil.getCellWidth(SheetUtil.java:185)
        at org.apache.poi.ss.util.SheetUtil.getColumnWidthForRow(SheetUtil.java:336)
        at org.apache.poi.ss.util.SheetUtil.getColumnWidth(SheetUtil.java:280)
        at org.apache.poi.ss.util.SheetUtil.getColumnWidth(SheetUtil.java:257)
        at org.apache.poi.xssf.streaming.SXSSFSheet.autoSizeColumn(SXSSFSheet.java:1621)
        at org.apache.poi.xssf.streaming.SXSSFSheet.autoSizeColumn(SXSSFSheet.java:1567)
        at io.quarkiverse.poi.it.POIResource.xlxs(POIResource.java:98)
        at io.quarkiverse.poi.it.POIResource_ClientProxy.xlxs(Unknown Source)
        at java.base@17.0.6/java.lang.reflect.Method.invoke(Method.java:568)
        at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:170)
        at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:130)
        at org.jboss.resteasy.core.ResourceMethodInvoker.internalInvokeOnTarget(ResourceMethodInvoker.java:660)
        at org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTargetAfterFilter(ResourceMethodInvoker.java:524)
        at org.jboss.resteasy.core.ResourceMethodInvoker.lambda$invokeOnTarget$2(ResourceMethodInvoker.java:474)
        at org.jboss.resteasy.core.interception.jaxrs.PreMatchContainerRequestContext.filter(PreMatchContainerRequestContext.java:364)
        at org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTarget(ResourceMethodInvoker.java:476)
        at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:434)
        at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:408)
        at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:69)
        at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:492)
        ... 17 more
Caused by: java.lang.reflect.InvocationTargetException
        at java.base@17.0.6/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
        at java.base@17.0.6/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
        at java.desktop@17.0.6/sun.font.FontManagerFactory$1.run(FontManagerFactory.java:85)
        ... 45 more
Caused by: java.lang.NullPointerException
        at java.desktop@17.0.6/sun.awt.FontConfiguration.getVersion(FontConfiguration.java:1264)
        at java.desktop@17.0.6/sun.awt.FontConfiguration.readFontConfigFile(FontConfiguration.java:224)
        at java.desktop@17.0.6/sun.awt.FontConfiguration.init(FontConfiguration.java:106)
        at java.desktop@17.0.6/sun.awt.X11FontManager.createFontConfiguration(X11FontManager.java:706)
        at java.desktop@17.0.6/sun.font.SunFontManager$2.run(SunFontManager.java:358)
        at java.desktop@17.0.6/sun.font.SunFontManager$2.run(SunFontManager.java:315)
        at java.base@17.0.6/java.security.AccessController.executePrivileged(AccessController.java:168)
        at java.base@17.0.6/java.security.AccessController.doPrivileged(AccessController.java:318)
        at java.desktop@17.0.6/sun.font.SunFontManager.<init>(SunFontManager.java:315)
        at java.desktop@17.0.6/sun.awt.FcFontManager.<init>(FcFontManager.java:35)
        at java.desktop@17.0.6/sun.awt.X11FontManager.<init>(X11FontManager.java:56)
        ... 48 more
```

--- a/README.md
+++ b/README.md
@@ -39,7 +39,7 @@ This extension provides the following features:
 ## Docker
 
 When building native images in Docker using the standard Quarkus Docker configuration files some additional features need to be
-installed to support Apache POI.  Specifically font information is not included in Red Hat's ubi-minimal images.  To install it
+installed to support Apache POI.  Specifically font information is not included in [Red Hat's ubi-minimal images](https://developers.redhat.com/products/rhel/ubi).  To install it
 simply add these lines to your `DockerFile.native` file:
 
 ```shell

--- a/README.md
+++ b/README.md
@@ -35,6 +35,21 @@ This extension provides the following features:
 
 - Native image support
 - Support for Apache POI 5.2.3
+
+## Docker
+
+When building native images in Docker using the standard Quarkus Docker configuration files some additional features need to be
+installed to support Apache POI.  Specifically font information is not included in Red Hat's ubi-minimal images.  To install it
+simply add these lines to your `DockerFile.native` file:
+
+```shell
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.7
+
+######################### Set up environment for POI #########################
+RUN microdnf update && microdnf install freetype && microdnf install fontconfig
+```
+
+
 ## Contributors ✨
 
 Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):

--- a/docs/modules/ROOT/pages/index.adoc
+++ b/docs/modules/ROOT/pages/index.adoc
@@ -20,3 +20,30 @@ For instance, with Maven, add the following dependency to your POM file:
     <version>{project-version}</version>
 </dependency>
 ----
+
+== Docker
+
+When building native images in Docker using the standard Quarkus Docker configuration files some additional features need to be
+installed to support Apache POI.  Specifically font information is not included in Red Hat's ubi-minimal images.  To install it
+simply add these lines to your `DockerFile.native` file:
+
+[source]
+----
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.7
+
+######################### Set up environment for POI ##########################
+RUN microdnf update && microdnf install freetype && microdnf install fontconfig
+######################### Set up environment for POI ##########################
+
+WORKDIR /work/
+RUN chown 1001 /work \
+    && chmod "g+rwX" /work \
+    && chown 1001:root /work
+COPY --chown=1001:root target/*-runner /work/application
+
+EXPOSE 8080
+USER 1001
+
+CMD ["./application", "-Dquarkus.http.host=0.0.0.0"]
+----
+

--- a/docs/modules/ROOT/pages/index.adoc
+++ b/docs/modules/ROOT/pages/index.adoc
@@ -24,7 +24,7 @@ For instance, with Maven, add the following dependency to your POM file:
 == Docker
 
 When building native images in Docker using the standard Quarkus Docker configuration files some additional features need to be
-installed to support Apache POI.  Specifically font information is not included in Red Hat's ubi-minimal images.  To install it
+installed to support Apache POI.  Specifically font information is not included in [Red Hat's ubi-minimal images](https://developers.redhat.com/products/rhel/ubi).  To install it
 simply add these lines to your `DockerFile.native` file:
 
 [source]

